spec/domain/people/merger_spec.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe People::Merger do
+
+  let(:person) { Fabricate(:person) }
+  let(:duplicate) { Fabricate(:person_with_address_and_phone) }
+  let(:actor) { people(:root) }
+
+  let(:merger) { described_class.new(@src_person.id, @dst_person.id, actor) }
+
+  before do
+    Fabricate('Group::BottomGroup::Member',
+              group: groups(:bottom_group_one_one),
+              person: duplicate)
+  end
+
+  context 'merge people' do
+
+    it 'copies attributes, removes source person, creates log entry' do
```

Comment:
Vielleicht aufteilen in mehrere `it`s in einem gemeinsamen `context` der den Setup macht